src/xss-filters.js

```diff
@@ -376,19 +381,21 @@ exports._getPrivFilters = function () {
         // The state machine in CSS 3.0 is more well defined - http://www.w3.org/TR/css-syntax-3/#consume-a-url-token0
         // CSS_UNQUOTED_URL = /['\(\)]/g; // " \ treated by encodeURI()   
         yceuu: function(s) {
-            return cssUrl(s, CSS_UNQUOTED_URL);
+            return cssUrl(s).replace(CSS_UNQUOTED_URL, function (chr) {
+                return  chr === '\''        ? '\\27 ' :
```

Comment:
I think either one is fine.
- `\\27 ` is never found being parsed as if it equals to a quote syntactically.
- `'` is not touched by `encodeURI()` and `encodeURIComponent()`, so just keep it intact.
- and the choice is inherited from your last commit.